src/Repository/PostRepository.php

```diff
@@ -82,20 +81,16 @@ public function findBySearchQuery(string $rawQuery, int $limit = Post::NUM_ITEMS
     }
 
     /**
-     * Removes all non-alphanumeric characters except whitespaces.
-     */
-    private function sanitizeSearchQuery(string $query): string
-    {
-        return trim(preg_replace('/[[:space:]]+/', ' ', $query));
-    }
-
-    /**
-     * Splits the search query into terms and removes the ones which are irrelevant.
+     * Transforms the search string into an array of search terms.
      */
     private function extractSearchTerms(string $searchQuery): array
     {
+        $searchQuery = trim(preg_replace('/[[:space:]]+/', ' ', $query));
```

Comment:
@o-alquimista, You forgot to replace a `$query` variable with the `$searchQuery`, that's why TavisCI fails.